### PR TITLE
Add support for `zcrypt` to Beaker plugin

### PR DESCRIPTION
**Pull request by happz**
Pull Request Checklist

* [x] implement the feature
* [x] extend the test coverage
* [x] mention the version
* [x] include a release note

### CODE REVIEW

File: tmt/steps/provision/mrack.py [hunk 1]
```diff
@@ -357,6 +358,42 @@ def _transform_virtualization_is_virtualized(
...
+                'system',
+                children=[MrackHWKeyValue('ZCRYPT_MODEL', beaker_operator, actual_value)])])
+
+    return MrackHWGroup(
+        'system',
+        children=[MrackHWKeyValue('ZCRYPT_MODEL', beaker_operator, actual_value)])
+
+
+def _transform_zcrypt_mode(
+        constraint: tmt.hardware.FlagConstraint,
```
Comment (psss) [inline tmt/steps/provision/mrack.py:380]:
```suggestion
        constraint: tmt.hardware.TextConstraint,
```

The `mode` key is not a boolean flag, is it?

**Review by psss — CHANGES_REQUESTED** [review]
Thanks, looks good, just one question.

File: tmt/steps/provision/mrack.py (hunk 1 above)
Comment (happz) [reply tmt/steps/provision/mrack.py]: Fixed & rebased.

**Review by psss — APPROVED** [review]
Thanks!

**Pull request merged**